### ISSUE TITLE
ProjectFile has a None option

### ISSUE THREAD

**Issue by mbauman** [invenia/PkgTemplates.jl#182]
This was a little funny:

```Julia
ProjectFile keywords to customize:
[press: d=done, a=all, n=none]
   [X] version
 > [X] None
```

With both selected, the `None` option seemed to win.   Seems like this `None` option should be removed.

**Comment by christopher-dG** [comment]
It's an implementation detail of TerminalMenus, you can't create a menu with just one entry. 🙁

**Comment by mbauman** [comment]
Ah, that's unfortunate.  Perhaps this should be an upstream issue?  Seems like a 1-element menu should make sense and be easy to support.

cf. https://github.com/JuliaLang/julia/pull/35915#issuecomment-638316817

**Comment by christopher-dG** [comment]
Yeah, I agree. Tbh we're probably better off using the non-stdlib version because the stdlib will never backport these kinds of changes to 1.0.

**Comment by tomyun** [comment]
FYI, single option menu for REPL.TerminalMenus just got merged (likely available with Julia 1.6+): JuliaLang/julia#36369

**Comment by christopher-dG** [comment]
I think some people have said that 1.6 has potential to be the next LTS, so when that comes we can make the change 🙂
